CRM/Core/BAO/SchemaHandler.php

```diff
@@ -758,7 +758,23 @@ public static function createMissingIndices($missingIndices) {
    */
   public static function buildFieldChangeSql($params, $indexExist) {
     $sql = str_repeat(' ', 8);
-    $sql .= "ALTER TABLE {$params['table_name']}";
+    $sql .= "ALTER TABLE {$params['table_name']} ";
+    return $sql . self::getFieldAlterSQL($params, $indexExist, $sql);
```

Comment:
third parameter in getFieldAlterSQL() needed? Can't see any 3rd parameter defined in the function

Reply:
indeed I later removed it but not in this PR - fixed